ecr-login/ecr.go

```diff
@@ -115,13 +115,28 @@ func (self ECRHelper) Get(serverURL string) (string, string, error) {
 		return "", "", credentials.NewErrCredentialsNotFound()
 	}
 
+	profile, err := config.GetRegistryProfile(serverURL)
```

Comment:
It may be overkill to try fetch the config file each time. I could potentially introduce a new env var (e.g. `AWS_ECR_USE_REGISTRY_CONFIG`) to determine whether to fetch?